### PR TITLE
Python line limit 88

### PR DESCRIPTION
**Pull request by SimonHeybrock**
Based on recent discussion, this is to try out extending Python line limit to 88. See https://black.readthedocs.io/en/stable/the_black_code_style/current_style.html#line-length for motivation. I reformated *some* files so we can have a look if we are happy with this config, please have a look.

### CODE REVIEW

**Comment by jl-wynen** [comment]
Works for my setup. Interesting to see how in some places yapf splits lines and in other it joins them. Overall, this looks better than before.
I would be fine with increasing the limit more. Did you see if it would make a big difference to go to 100 or so?

**Comment by SimonHeybrock** [comment]
> Works for my setup. Interesting to see how in some places yapf splits lines and in other it joins them. Overall, this looks better than before.
> I would be fine with increasing the limit more. Did you see if it would make a big difference to go to 100 or so?

Looking at `formatting_html.py`, reduces by 19 lines when we go to 88, and by a further 15 lines when going to 100. See https://github.com/scipp/scipp/pull/2072/files. My feeling: Better in some cases, but some lines start to look like spaghetti!

**Comment by Tom-Willemsen** [comment]
I'm generally in favour of increasing the line-length limit. Of the 3 examples you've provided I like 100 best, followed by 88 and then the current limit.

**Comment by Tom-Willemsen** [comment]
One thing to think about though - we should try to apply this change when there aren't too many large pending changes in python code, otherwise we'll create massive amounts of git conflicts that will need resolving.

**Comment by nvaytet** [comment]
@SimonHeybrock how does this play with your work setup where you tend to have quite a few (usually not very wide) editors next to each other? Does the line length become a problem? especially with 100?

I think i like 88 for now.
I don't really mind have more lines in files.

**Comment by SimonHeybrock** [comment]
> @SimonHeybrock how does this play with your work setup where you tend to have quite a few (usually not very wide) editors next to each other? Does the line length become a problem? especially with 100?

I think 88 is fine for that. With 100 more screen space is wasted, because most lines are shorter.

> I think i like 88 for now.
> I don't really mind have more lines in files.

I also think I would go with that, since it is "standard".

File: setup.cfg [hunk 1]
```diff
@@ -0,0 +1,8 @@
+[flake8]
+# See https://black.readthedocs.io/en/stable/the_black_code_style/current_style.html#line-length
+max-line-length = 88
+extend-ignore = E203
+
+[yapf]
+based_on_style = pep8
+column_limit = 88
```
Comment (OwenArnold) [inline setup.cfg:8]: 👍  Looks better to me

**Comment by nvaytet** [comment]
I'm interested in seeing the changes in some of the test files.

**Comment by SimonHeybrock** [comment]
> I'm interested in seeing the changes in some of the test files.

Already pushed those changes.

**Pull request merged**